lib/sensu-plugins-windows/version.rb

```diff
@@ -1,7 +1,7 @@
 module SensuPluginsWindows
   module Version
     MAJOR = 2
-    MINOR = 4
+    MINOR = 5
```

Comment:
per previous comment generally the versioning happens by maintainers post acceptance prior to release.